lib/nostrum/api.ex

```diff
@@ -51,6 +51,7 @@ defmodule Nostrum.Api do
   alias Nostrum.Struct.Channel
   alias Nostrum.Struct.Emoji
   alias Nostrum.Struct.Guild.{Member, Role}
+  alias Nostrum.Struct.Invite
```

Comment:
We should probably consolidate and nest alias here to correct some credo issues.